packages/cyberstorm/src/components/Toast/Provider.tsx

```diff
@@ -37,16 +42,16 @@ const toastReducer = (
 };
 
 interface ContextInterface {
-  addToast: (props: Omit<ToastProps, "id">) => void;
+  addToast: ({ ...props }: Omit<ToastProps, "id">) => void;
   remove: (id: string) => void;
 }
 
-const ToastContext = createContext<ContextInterface | null>(null);
+export const ToastContext = createContext<ContextInterface | null>(null);
```

Comment:
Seems `ToastContext` is used only within this file so why export it? I'd assume we'd like the developers to use the `useToast` hook so not providing the context might be a good thing. 🤷🏻